BGAnimations/ScreenGameplay underlay/PerPlayer/SubtractiveScoring.lua

```diff
@@ -65,14 +95,28 @@ bmt.InitCommand=function(self)
 end
 
 bmt.JudgmentMessageCommand=function(self, params)
-	if player == params.Player then
+	if player == params.Player and not mods.ShowEXScore then
 		tns = ToEnumShortString(params.TapNoteScore)
 		hns = params.HoldNoteScore and ToEnumShortString(params.HoldNoteScore)
 		self:queuecommand("SetScore")
 	end
 end
 
 
+bmt.ExCountsChangedMessageCommand=function(self, params)
+	if player == params.Player and mods.ShowEXScore then
+		local actual = params.ExScore
+		local possible = GetPossibleExScore(params.ExCounts)
+		local score = possible - actual
+		
+		-- handle floating point equality.
+		if score <= 0.0001 then
```

Comment:
This should be
```lua
if score > 0.0001 then
```
otherwise the EX score never shows up.

Reply:
AH woops fixed.